packages/wrangler/src/d1/create.tsx

```diff
@@ -53,7 +53,6 @@ export const Handler = withConfig<HandlerOptions>(
 				},
 				body: JSON.stringify({
 					name,
-					experimental: true,
```

Comment:
the API no longer requires this - all created DBs have been using the new backend for about 8 months now